manager/middlewares/authenticator_test.go

```diff
@@ -0,0 +1,104 @@
+package middlewares
+
+import (
+	"app/base/models"
+	"app/base/utils"
+	"app/test"
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	identityMock = identity.XRHID{Identity: identity.Identity{
+		Type:  "User",
+		OrgID: "3340851",
+		User: identity.User{
+			Username:  "unit@test.com",
+			Email:     "unit@test.com",
+			FirstName: "Unit",
+			LastName:  "Test",
+			Active:    true,
+			OrgAdmin:  false,
+			Internal:  false,
+			Locale:    "en_US",
+			UserID:    "1337",
+		},
+		Internal: identity.Internal{
+			OrgID: "3340851",
+		},
+	}}
+)
+
+var authenticator gin.HandlerFunc
+
+func TestAuthenticatorValid(t *testing.T) {
+	// valid account in db
+	identityMock.Identity.AccountNumber = "13"
```

Comment:
Would probably be better to define this kind of thing as constants in the `test` package

Reply:
I want to fix the testing data, to add more and merge the accounts to single one (probably to ID = 0 to keep it simple), I will change this in other PR.